nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/nuxeo-ai.html

```diff
@@ -76,333 +76,343 @@
 </nuxeo-slot-content>
 <nuxeo-slot-content name="enrichmentRemoveAction" slot="RESULTS_ACTIONS" order="-90">
   <template>
-    <nuxeo-ai-bulk-remove-button provider="[[nxProvider]]"></nuxeo-ai-bulk-remove-button>
+    <nuxeo-filter user="[[user]]"
+                  expression="user.isAdministrator || user.properties.groups.indexOf('insight-managers') >= 0">
```

Comment:
I guess it's the only line that was changed?

Reply:
Oh yeah im sorry for the format, those lines only indeed